desafio-02/robsonalexandre/bash/primos.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+num=1
+while ((num <= 10000)); do
```

Comment:
Voce pod simplificar esse loop com:
`for num in {1..10000}; do (...)`

Ou melhor ainda, como voce sabe que 1 não é primo, comece o loop acima com 2 ao invés de 1.

Reply:
Realmente, como os limites de início e final do loop são definidos, simplifica o código.
Sugestões foram implementadas no código.